README.md

```diff
@@ -380,21 +380,40 @@ One consideration in the short term is that these calls may have different
 associated [debug modes or keys](#temporary-debugging-mechanism). In this case,
 only calls sharing those details should be batched together.
 
-#### Contributions limit
+#### Limiting the number of contributions per report
 
 We will also need a limit on the number of contributions within a single report.
 In the case that too many contributions are specified with a ‘batching scope’,
-we should truncate them to the limit.
-
-However, to reduce the impact of this limit, we will pre-aggregate (i.e. merge)
-any contributions that have the same bucket and [filtering
+we should truncate them to the limit. To reduce the impact of this limit, we
+will merge any contributions that have the same bucket and [filtering
 ID](https://github.com/patcg-individual-drafts/private-aggregation-api/blob/main/flexible_filtering.md#proposal-filtering-id-in-the-encrypted-payload)
 before truncation.
 
-If necessary, we could instead split the contributions back into multiple
-reports, each respecting the limit.
+There is a tradeoff to consider &mdash; larger reports are more costly for the
+aggregation service to process, but smaller reports have lower utility. There
+are several approaches to selecting a limit.
+
+- *Default limit:* The simplest strategy is to use one limit for all reports.
+  Our implementation currently enforces a limit of 20 contributions per report.
+
+- *Default limit per API:* The limit may be determined based on the calling API.
+  In particular, Protected Audience reports may benefit from a higher limit more
+  than Shared Storage reports. Our implementation plan is to set the limit at 20
+  contributions per report for Shared Storage and 100 contributions per report
+  for Protected Audience.
+
+- *Per-context limit:* Callers may request a different limit on each isolated
```

Comment:
AKA "eager report sizing" and "per-caller sizing," which I think is a little ambiguous relative to the previous bullet.